stream_framework/serializers/activity_serializer.py

```diff
@@ -40,12 +43,15 @@ def loads(self, serialized_activity):
         actor_id, verb_id, object_id, target_id = map(
             int, parts[:4])
         activity_datetime = epoch_to_datetime(float(parts[4]))
-        pickle_string = str(parts[5])
+        # just in case the original pickle_string contains a comma
+        pickle_string = ','.join(parts[5:])
```

Comment:
If you know how many parts there should be, [str.split()](https://docs.python.org/3/library/stdtypes.html#str.split) takes a second 'maxsplit' parameter, which might avoid the need to rejoin like this: `serialized_activity.split(',', 4)`